herald/herald/Sensor/BLE/BLEDatabase.swift

```diff
@@ -320,38 +320,59 @@ class BLEPseudoDeviceAddress {
         }}
     
     init?(fromAdvertisementData: [String: Any]) {
-        guard let manufacturerData = fromAdvertisementData["kCBAdvDataManufacturerData"] as? Data else {
+        guard let manufacturerData = fromAdvertisementData[CBAdvertisementDataManufacturerDataKey] as? Data else {
```

Comment:
Please double check which iOS versions this constant is available in

Reply:
iOS 5.0+ 👍